src/Utility/include/OpenLoco/Utility/Stream.hpp

```diff
@@ -205,9 +209,9 @@ namespace OpenLoco
         bool _writing{};
 
     public:
-        FileStream(const std::filesystem::path path, uint8_t flags)
+        FileStream(const std::filesystem::path path, StreamFlags flags)
         {
-            if (flags & StreamFlags::write)
+            if (flags == StreamFlags::write)
```

Comment:
Is this actually correct? If it can be only in either read or write mode then I wouldn't use flags for this.

Reply:
Yeah i think we should perhaps just change these to a plain enum class of `StreamMode`

Reply:
Will do